sssd_test_framework/roles/client.py

```diff
@@ -53,6 +54,11 @@ def __init__(self, *args, **kwargs) -> None:
         Call commands from sssctl.
         """
 
+        self.realmd: RealmdUtils = RealmdUtils(self.host)
```

Comment:
Like the command, maybe we call this self.realm ?

Reply:
added